dcrjson/walletsvrcmds.go

```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
 	}
 }
 
+// FundRawTransactionOptions represents the optional inputs to fund
+// a raw transaction.
+type FundRawTransactionOptions struct {
+	ChangeAccount         *string
+	LockUnspents          *bool `jsonrpcdefault:"false"`
```

Comment:
Let's not add this for now.  Locking outputs doesn't actually work in dcrwallet currently.